src/xenia/base/system_gnulinux.cc

```diff
@@ -29,7 +33,19 @@ void LaunchWebBrowser(const std::string_view url) {
   system(cmd.c_str());
 }
 
-void LaunchFileExplorer(const std::filesystem::path& path) { assert_always(); }
+void LaunchFileExplorer(const std::filesystem::path& path) {
+  pid_t pid;
+  int n = path.string().length();
```

Comment:
Implicit truncation from `size_t` to `int`.

Reply:
`n` is not a good variable name choice also, as well as `cmdxdg` and `cmdpath` — shortenings are generally not recommended unless the full word is very long, and words should be separated with an underscore.

Reply:
Right, Realised `n` wouldn't even be necessary anyway, Will remove that and also change the name of the other two variables.